src/components/trends/common/Tooltip.css

```diff
@@ -0,0 +1,40 @@
+.tooltip {
+  display: block;
+  font-size: 14px;
+  background-color: #FFFFFF;
+  border: solid;
+  position: absolute;
+  border-radius: 4px;
+  opacity: 0.75;
+}
+
+.content {
+  display: flex;
```

Comment:
I can't remember if we had some flexbox styles separated out for composition in layout.css. Perhaps check?